tests/hazmat/primitives/test_ec.py

```diff
@@ -1068,9 +1068,9 @@ def test_key_exchange_with_vectors(self, backend, vector):
                 ec._CURVE_TYPES[vector['curve']]()
             )
         )
-        # Errno 5 and 6 indicates a bad public key, this doesn't test the ECDH
-        # code at all
-        if vector['fail'] and vector['errno'] in [5, 6]:
+        # Errno 5-7 indicates a bad public or private key, this doesn't test the
```

Comment:
This line is longer than our 80 (79?) char max. Can you please wrap it so the linter is happy :-)

Reply:
Done. (Seems to be 79.)